packages/kolibri-common/components/SearchFiltersPanel/index.vue

```diff
@@ -21,94 +21,93 @@
       v-if="windowIsLarge || !currentCategory"
       :class="windowIsLarge ? '' : 'drawer-panel'"
     >
-      <div v-if="Object.keys(availableLibraryCategories).length">
```

Comment:
It seems this check is not being persisted anywhere.

If you look at the original PR where this was moved, it originally was used to wrap the section containing the categories, so that if no categories are in the local library, the section does not get displayed at all.